cw_bitcoin/lib/electrum.dart

```diff
@@ -243,22 +245,13 @@ class ElectrumClient {
       });
 
   Future<String> broadcastTransaction(
-      {required String transactionRaw, BasedUtxoNetwork? network}) async {
-    if (network == BitcoinNetwork.testnet) {
-      return http
-          .post(Uri(scheme: 'https', host: 'blockstream.info', path: '/testnet/api/tx'),
-              headers: <String, String>{'Content-Type': 'application/json; charset=utf-8'},
-              body: transactionRaw)
-          .then((http.Response response) {
-        if (response.statusCode == 200) {
-          return response.body;
-        }
-
-        throw Exception('Failed to broadcast transaction: ${response.body}');
-      });
-    }
```

Comment:
why remove testnet api?

Reply:
Not needed anymore to send txs, and it was used to log the response but new error messages by id can be used instead